build.sbt

```diff
@@ -212,7 +214,13 @@ lazy val server: Project = (project in file("server"))
       credentials += Credentials(Path.userHome / ".s3_elasticmq_credentials"),
       // docker
       dockerExposedPorts := Seq(9324,9325),
-      dockerBaseImage := "openjdk:8u212-b04-jdk-stretch",
+      dockerBaseImage := "openjdk:11-jdk-stretch",
+      dockerBuildOptions := dockerBuildOptions.value :+ "--platform=linux/arm64,linux/amd64" :+ "--push",
+      dockerBuildCommand := {
+        val old = dockerBuildCommand.value
+        val withBuildx = old.take(1) ++ Seq("buildx") ++ old.drop(1)
```

Comment:
some comments would be good explaining why we replace that element - what is the structure of the command.

also, this assumes that the returned list has always 3 elements. Wouldn't it be safer to replace a specific element, and fail if there are e.g. 4 elements?

Reply:
Added comment, should be clear enough. The returned list will not have always 3 elements. It may be more or less and it's safe. We only need command `buildx` after first `[dockerExecCommand]` which is docker by default and there is no need to modify that exec command.

Reply:
ah ok, you want to add, not replace an element. Clear now with the comment :)